packages/infra/lib/api-stack/ccda-search-connector.ts

```diff
@@ -0,0 +1,169 @@
+import { Duration } from "aws-cdk-lib";
+import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import { EbsDeviceVolumeType, IVpc } from "aws-cdk-lib/aws-ec2";
+import { IFunction, ILayerVersion } from "aws-cdk-lib/aws-lambda";
+import { SqsEventSource } from "aws-cdk-lib/aws-lambda-event-sources";
+import { IDomain } from "aws-cdk-lib/aws-opensearchservice";
+import * as s3 from "aws-cdk-lib/aws-s3";
+import { IQueue } from "aws-cdk-lib/aws-sqs";
+import { Construct } from "constructs";
+import { getConfig, METRICS_NAMESPACE } from "../shared/config";
+import { createLambda as defaultCreateLambda } from "../shared/lambda";
+import OpenSearchConstruct, { OpenSearchConstructProps } from "../shared/open-search-construct";
+import { createQueue as defaultCreateQueue, provideAccessToQueue } from "../shared/sqs";
+
+export function settings(): {
+  openSearch: Omit<OpenSearchConstructProps, "region" | "vpc"> & { indexName: string };
+  connectorName: string;
+  lambda: {
+    memory: number;
+    batchSize: number;
+    maxConcurrency: number;
+    timeout: Duration;
+  };
+  sqs: {
+    maxReceiveCount: number;
+    visibilityTimeout: Duration;
+    delayWhenRetrying: Duration;
+  };
+} {
+  // TODO 1050 once this works well in staging, we can attempt a smaller/cheaper setup for it, but first validate
+  // the prod one, so we don't test prod only when releasing there.
+  // const config = getConfig();
+  // const isLarge = isProd(config) || isSandbox(config);
+  const isLarge = true;
+  // How long can the lambda run for, max is 900 seconds (15 minutes)
+  const timeout = Duration.minutes(5);
+  return {
+    // https://docs.aws.amazon.com/opensearch-service/latest/developerguide/supported-instance-types.html
+    openSearch: {
+      capacity: {
+        dataNodes: isLarge ? 2 : 1,
+        dataNodeInstanceType: isLarge ? "t3.small.search" : "t3.small.search",
+        masterNodes: isLarge ? undefined : undefined, // odd number, 3+; when not set this is done by data nodes
+        masterNodeInstanceType: isLarge ? undefined : undefined,
```

Comment:
intentional to be undefined?

Reply:
Yeah, same as the other one, keeping it small to star with and we see how it goes.

When there's no master index, the nodes take care of it - which is bad under a scenario where there's a lot of load.